towncrier_template.rst

```diff
@@ -0,0 +1,27 @@
+{% for section, _ in sections.items() %}
+{% set underline = underlines[0] %}{% if section %}{{section}}
+{{ underline * section|length }}{% set underline = underlines[1] %}
+{% endif %}
+{% if sections[section] %}
+{% for category, val in definitions.items() if category in sections[section]%}
+{% if definitions[category]['showcontent'] %}
```

Comment:
It looks like the category names aren't used in the template; is there any plan to separate the different types of changes in the generated changelog entries?

Reply:
Possibly, but for now the categories are just used for ordering purposes so that the important stuff is sorted on top.